sevntu-checks/src/main/sh/ast.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 #################################################################################
-#  This is a useful script which builds Checkstyle AST (abstract synthax tree)  #
+#  This is a useful script which builds Checkstyle AST (abstract syntax tree)  #
```

Comment:
please make the final `#` on this line, line up with the other lines.

Reply:
Done